### PR TITLE
Adding return statuses for PushObject and PullObject (MCS-62, MCS-63).

### PR DESCRIPTION
**Pull request by rartiss55**
Python API changes here: https://github.com/NextCenturyCorporation/MCS/pull/18

### CODE REVIEW

File: unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs [hunk 1]
```diff
@@ -2086,10 +2086,18 @@ public void ChangeColorOfMaterials(ServerAction action)
         }
 
         public virtual void PushObject(ServerAction action) {
+            if (!physicsSceneManager.UniqueIdToSimObjPhysics.ContainsKey(action.objectId)) {
+                errorMessage = "Object ID appears to be invalid.";
+                actionFinished(false);
```
Comment (brianpippin) [inline unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs]: Should these have the "Debug.Log(errorMessage);" that most of the other functions have, I noticed it was missing in another place as well.

Comment (rartiss55) [reply unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs]: Ah thank you! I probably quickly copy + pasted from someplace that didn't have it :)

File: unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs [hunk 2]
```diff
@@ -2236,6 +2256,7 @@ public void DirectionalPush(ServerAction action)
                 errorMessage = "Target must be SimObjPhysics!";
                 Debug.Log(errorMessage);
                 actionFinished(false);
+                this.lastActionStatus = Enum.GetName(typeof(ActionStatus), ActionStatus.FAILED);
```
Comment (ThomasSchellenbergNextCentury) [inline unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs]: @rartiss55 I think you can return NOT_PICKUPABLE here.

File: unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs [hunk 3]
```diff
@@ -2248,12 +2269,14 @@ public void DirectionalPush(ServerAction action)
...
                 errorMessage = "Target Sim Object cannot be moved. It's primary property must be Pickupable or Moveable";
                 actionFinished(false);
+                this.lastActionStatus = Enum.GetName(typeof(ActionStatus), ActionStatus.NOT_PICKUPABLE);
                 return;
             }
 
             if (!action.forceAction && target.isInteractable == false) {
                 errorMessage = "Target is not interactable and is probably occluded by something!";
                 actionFinished(false);
+                this.lastActionStatus = Enum.GetName(typeof(ActionStatus), ActionStatus.FAILED);
```
Comment (ThomasSchellenbergNextCentury) [inline unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs]: @rartiss55 You can return NOT_PICKUPABLE here as well.

**Comment by ThomasSchellenbergNextCentury** [comment]
This looks good now.  My only other thought is about returning FAILED for trying to Push or Pull held objects.  I'm not sure whether it's correct to fail, or whether we should just redirect to Throw for them.  We can worry about that one later, though.

**Pull request merged**